src/redisearch_rs/rlookup/src/lookup.rs

```diff
@@ -356,6 +356,7 @@ impl<'a> RLookupKey<'a> {
         unsafe { Pin::new_unchecked(b) }
     }
 
+    #[cfg(test)]
```

Comment:
```suggestion
    #[cfg(any(debug_assertions, test))]
```
I think you want this rather because of this call

https://github.com/RediSearch/RediSearch/blob/5cef3fbd9311eb7c37c53f10dd80304cd3ccf1e7/src/redisearch_rs/rlookup/src/lookup.rs#L390-L398

Reply:
`is_tombstone` is also used in in another test `keylist_override_key_iterate`

https://github.com/RediSearch/RediSearch/blob/5cef3fbd9311eb7c37c53f10dd80304cd3ccf1e7/src/redisearch_rs/rlookup/src/lookup.rs#L1388-L1408

Reply:
My suggestion will cover both right?

Reply:
Yes